packages/data/src/world/mq/jabu_jabu_mq.yml

```diff
@@ -237,7 +232,7 @@
 "Jabu-Jabu Pre-Boss Room":
   dungeon: JJ
   exits:
-    "Jabu-Jabu Main Elevator Room Ledge Before Pre-Boss After Red Tentacle": "true"
+    "Jabu-Jabu Main Elevator Room Ledge Before Pre-Boss": "event(JABU_TENTACLE_RED)"
```

Comment:
This isn't needed. Just leave it true. The tentacle isn't on this side. And even when you come through the other side you just get whacked to the front anyway, which is why it needs to connect directly to Ledge Before Pre-Boss.

Reply:
Fixed.